hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/WALEntryStream.java

```diff
@@ -256,7 +256,6 @@ private void dequeueCurrentLog() throws IOException {
     logQueue.remove(walGroupId);
     setCurrentPath(null);
     setPosition(0);
-    metrics.decrSizeOfLogQueue();
```

Comment:
We already decrement the size of logqueue metric at line#256 on logQueue#remove(walGroupId) so no need to decrement again.

Reply:
I see it in here...

```
  public void remove(String walGroupId) {
    PriorityBlockingQueue<Path> queue = getQueue(walGroupId);
    if (queue == null || queue.isEmpty()) {
      return;
    }
    queue.remove();
    // Decrease size logQueue.
    this.metrics.decrSizeOfLogQueue();
    // Re-compute age of oldest wal metric.
    this.metrics.setOldestWalAge(getOldestWalAge());
  }
```

Reply:
Yes thats correct. On line#256 we call this method only.